src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd

```diff
@@ -422,6 +422,7 @@
             <xsd:enumeration value="1" />
             <xsd:enumeration value="0" />
             <xsd:enumeration value="allow_no_handlers" />
+            <xsd:enumeration value="throw_no_routing" />
```

Comment:
I know, what I mean is shouldn't we rename that param to allow_no_senders, which would default to true, and that one could turn to false? I'm challenging the name because naming is hard :) maybe your proposal is the good one :shrug: 
throw_no_routing, throw_no_senders, allow_no_senders, etc / anyone?

Reply:
With the default to `true` or with a name like `allow_no_senders`, we've got a bc break.

Reply:
For soncistency with `allow_no_handlers`, I still prefer `allow_no_senders` as a name, with `true` as default (which is the current behavior IIUC, aka no BC break).